actionpack/lib/system_test_case.rb

```diff
@@ -0,0 +1,57 @@
+require "system_testing/test_helper"
+require "system_testing/driver_adapter"
+
+module Rails
```

Comment:
If we're shipping this under the `actionpack` gem, should we use the `ActionPack` base module? 😬